x-pack/plugins/security_solution/public/detection_engine/rule_exceptions/components/flyout_components/add_to_rules_table/index.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React, { useEffect, useMemo, useState } from 'react';
+import type { CriteriaWithPagination } from '@elastic/eui';
+import { EuiSpacer, EuiPanel, EuiText, EuiInMemoryTable, EuiLoadingContent } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+import * as myI18n from './translations';
+import type { Rule } from '../../../../../detections/containers/detection_engine/rules/types';
+import { useFindRules } from '../../../../../detections/pages/detection_engine/rules/all/rules_table/use_find_rules';
+import { getRulesTableColumn } from '../utils';
+
+interface ExceptionsAddToRulesComponentProps {
+  initiallySelectedRules?: Rule[];
+  onRuleSelectionChange?: (rulesSelectedToAdd: Rule[]) => void;
+}
+
+const ExceptionsAddToRulesTableComponent: React.FC<ExceptionsAddToRulesComponentProps> = ({
+  initiallySelectedRules,
+  onRuleSelectionChange,
+}): JSX.Element => {
```

Comment:
I believe by default the functional component returns JSX 
So we can emit the`JSX.Element`